lib/support/cli.js

```diff
@@ -236,11 +236,17 @@ module.exports.parseCommandLine = function parseCommandLine() {
       config: 'config'
     })
     .option('viewPort', {
-      describe: 'Size of browser window WIDTHxHEIGHT'
+      describe: 'Size of browser window WIDTHxHEIGHT',
+      default: '1500x1200'
```

Comment:
This will always set the window size to 1500x1200, even when not running xvfb. Do we want that as a side effect? Maybe better to just apply it when running xvfb.